app/server.py

```diff
@@ -57,17 +53,17 @@ def search(search_engine):
             err = [404, 'No response', qformat]
             return bad_request(err)
 
-        if((db['queries'].find({query: query}).limit(1)) is False):
+        if db['queries'].find({query: query}).limit(1) is False:
```

Comment:
Same

Reply:
It merely more Pythonic.  It is not JavaScript or Lisp.